packages/expo-auth-session/src/providers/Google.ts

```diff
@@ -142,8 +142,9 @@ class GoogleAuthRequest extends AuthRequest {
 
     // Apply the default scopes
     const scopes = applyRequiredScopes(config.scopes, settings.minimumScopes);
-    const isImplicit =
-      config.responseType === ResponseType.Token || config.responseType === ResponseType.IdToken;
+    const responseType = config.responseType.split(' ');
+    const isImplicit = responseType.includes(ResponseType.Token) || responseType.includes(ResponseType.IdToken)
```

Comment:
```suggestion
    const responseTypes = config.responseType.split(' ');
    const isImplicit = responseTypes.includes(ResponseType.Token) || responseTypes.includes(ResponseType.IdToken)
```

Let's reflect in variable name that it's an array.